javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,19 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var checkStatus;
+  var i = array.length;
+  if (i > 0) {
+    while (i > 0) {
+      if (array[i - 1] === 7) {
+        checkStatus = true;
+        break;
+      } else {
+        checkStatus = false;
+        i--;
+      }
+    }
+  } else {
+    checkStatus = false;
+  }
```

Comment:
Same feedback here, there's no real reason this loop needs to be going backwards, please make it move forwards through the array.